src/odemis/acq/align/autofocus.py

```diff
@@ -159,12 +159,12 @@ def AcquireNoBackground(det, dfbkg=None, timeout=None):
     """
     if dfbkg is not None:
         # acquire background
-        bg_image = getNextImage(det, timeout)
+        bg_image = getNextImage(det, timeout)  # TODO code inspection says nothing returned, not sure...
```

Comment:
Well, it's just wrong. It's because in `getNextImage()` the variable is first initialised at None, but it's never the final value.